src/scmdata/ensemble.py

```diff
@@ -0,0 +1,226 @@
+"""
+:class:`ScmEnsemble` provides a container for holding data from different model runs.
+"""
+
+import warnings
+
+import numpy as np
+import pandas as pd
+
+from scmdata.plotting import inject_plotting_methods
+from scmdata.run import BaseScmRun
+
+
+class ScmEnsemble:
+    """
+    Container for holding multiple :obj:`ScmRun` objects
+    """
+
+    meta_col = "run_id"
+    """
+    str: Name of meta containing a unique identifer for each run
+    
+    If no value is provided, then an implicit value is create. This ensures that every timeseries
+    has unique metadata
+    """
+
+    def __init__(self, runs=None):
+        """
+
```

Comment:
Are we intending on adding more loading functionality? If no, we should change the notebook

Reply:
Just looked at the notebook properly, assuming we're waiting to sort this before we start mucking around with writing the notebook?